doc/installation_mariadb.md

```diff
@@ -0,0 +1,145 @@
+# Installation Guide for MariaDB
+
+VIDEX supports the following installation methods:
+
+1. Compile a complete MariaDB Server (including VIDEX engine)
+2. Use Docker image installation method
+
+## 1. Prepare Environment
+
+If you can configure the **Clang build environment** for MariaDB without Docker, you may skip to the next section.  
+Otherwise, please use Docker.
+
+
+### 1.1 Build Docker Image
+
+```bash
+docker build -t mariadb11_build -f build/Dockerfile.mariadb .
```

Comment:
Is there any difference between the environments for `mariadb_build` and `mySQL_build`?